services/resources/deployment.go

```diff
@@ -0,0 +1,63 @@
+package deployment
```

Comment:
from what @marstr mentioned this was a manual maintained package. Maybe create an issue to investigate whats up with it.

Reply:
I create #877 to track this.